src/base/base.ts

```diff
@@ -200,8 +204,10 @@ export default class Base {
     updates: ObjectType,
     key: string
   ): Promise<UpdateResponse> {
-    const trimmedKey = key.trim();
-    if (!trimmedKey.length) {
+    const trimmedKey = key?.trim();
+    if (!trimmedKey) {
+      throw new Error('Key is empty');
+    } else {
       throw new Error('Key is empty');
     }
```

Comment:
Remove the else block we don't need those anymore

Reply:
Fixed